nncf/experimental/openvino_native/graph/model_transformer.py

```diff
@@ -44,14 +45,19 @@ def transform(self, transformation_layout: TransformationLayout) -> ov.Model:
         :param transformations: lisf of the TransformationCommand transformations.
         """
         output_insert_transformations = []
+        quantizer_insert_transformations = []
```

Comment:
following the name below `_apply_quantizer_insertion_transformations`, you should call it like `quantizer_insertion_transformations`.

Reply:
fixed